packages/components/psammead-brand/README.md

```diff
@@ -47,7 +47,6 @@ The `scriptLink` can be used to render [ScriptLink](https://github.com/bbc/psamm
 | borderTop | Boolean | no | `false` | `true` |
 | borderBottom | Boolean | no | `false` | `true` |
 | scriptLink | Node | no | `null` | `<ScriptLink service='news' script={latin} href='https://www.bbc.com/serbian/lat'> Lat </ScriptLink>` |
-| dir | string | No | `'ltr' | One of `'rtl'` `'ltr' |
```

Comment:
Can you remember why were we using the `dir` prop before?

Reply:
We were using it to position the `ScriptLink`  `left: 0` and `right:0`
https://github.com/bbc/psammead/pull/2539/files#diff-4f91a28dd0d1d26145e763893261c522L188